recenter_tc/plugins/modules/sector_adjusters/recenter_tc.py

```diff
@@ -34,6 +33,14 @@
 name = "recenter_tc"
 
 
+def print_area_def(area_def, print_str):
+    """Print the supplied pyresample area definition using log with emphasis."""
+    messages = [print_str, str(area_def)]
+    for key, value in area_def.sector_info.items():
+        messages.append(f"{key}: {value}")
+    log_with_emphasis(LOG.info, *messages)
```

Comment:
This looks good for now. Eventually, I think this function should be part of the Sector plugin class but let's get this merged now to fix the problem and create a new issue addressing this like we talked about in today's meeting.

Once the issues has been created, please resolve this comment and merge.

Reply:
https://github.com/NRLMMD-GEOIPS/recenter_tc/issues/34

Going to merge this, so we can tag 1.12.2a0